client/src/pages/Dashboard/MediaRenderer.jsx

```diff
@@ -23,16 +23,15 @@ const VideoRenderer = ({ fileUrl, numberOfMedia, fileType }) => (
     </video>
   );
 
-const ImageRenderer = ({ props }) => {
-  const { fileUrl, numberOfMedia } = props;
+const ImageRenderer = ({ fileUrl, numberOfMedia  }) => {
   return <img src={fileUrl} alt="media of workout" className={`${dashboardStyles.image} ${numberOfMedia === 1 ? dashboardStyles.full : ''}`} />;
 };
 
 const MediaRenderer = (props) => {
   const fileType = getFileTypeFromUrl(props.fileUrl);
   const isVideo = fileType === 'video';
 
-  return <>{isVideo ? <VideoRenderer props={props} /> : <ImageRenderer props={props} />}</>;
+  return <>{isVideo ? <VideoRenderer {...props} /> : <ImageRenderer {...props} />}</>;
```

Comment:
There was inconsistency in how props was being destructured which led to the Videos being incorrectly rendered. The video renderer was using props destructuring while the Image rendering was not. Updated the code for both to destructure and now the variables (fileUrl, numberOfMedia, etc) will not yield undefined.